apps/widget-e2e/src/describer/modelAssessment/modelAssessmentDatasets.ts

```diff
@@ -129,7 +129,47 @@ const modelAssessmentDatasets = {
   DiabetesDecisionMaking: {
     cohortDefaultName: "All data",
     featureImportanceData: {
-      hasFeatureImportanceComponent: false
+      hasFeatureImportanceComponent: false,
+      datapoint: 89
+    },
+    whatIfCounterfactualsData: {
+      checkForClassField: false,
+      columnHeaderAfterSort: "s5",
+      columnHeaderBeforeSort: "age",
+      CreateYourOwnCounterfactualInputFieldUpdated: "25",
```

Comment:
can we follow consistant camelCase here?

Reply:
Good catch! updated to camel case.